### DISCUSSION TITLE
The features of premium are not made clear.

### DISCUSSION THREAD

**Discussion by UnBeatWaterGH** [AssetRipper/AssetRipper#1297] [Q&A]
Are there only 3 features?

**Comment by ds5678** [comment]
Yes, that is the current number of premium features. It will expand over time, but that's how many are offered right now.

**Comment by UnBeatWaterGH** [comment]
I'd recommend creating a page or list of features that makes it clear EXACTLY what features exist. Finding a Patreon post titled "Premium Launch Features" doesn't exactly make me think "These are all the features", especially as I can not view the premium changelogs as I do not pay you money.
